scripts/test_api_calls.py

```diff
@@ -0,0 +1,145 @@
+import time
+import json
+from helper_functions import *
+
+node_id     = ["1112a4d4-ffff-4ece-b3f0-8d36e3d85001", "1112a4d4-ffff-4ece-b3f0-8d36e3d85002",]
+subnet_id   = "8182a4d4-ffff-4ece-b3f0-8d36e3d88002"
+network_id  = "9192a4d4-ffff-4ece-b3f0-8d36e3d88001"
+project_id  = "3dda2801-d675-4688-a63f-dcda8d327f50"
+tenant_id   = "3dda2801-d675-4688-a63f-dcda8d327f50"
+security_group_id ="3dda2801-d675-4688-a63f-dcda8d111111"
+device_id="8182a4d4-ffff-4ece-b3f0-8d36e3d00003" 
+port_id = ["7122a4d4-fffm-5eee-b3f0-8d36e3d00106","7122a4d4-fffn-5eee-b3f0-8d36e3d00106"]
+ip_addrs =["10.0.1.101","10.0.1.102"]
+ip_mac_db = {}
+
+def post_httprequest(url, data=""):
+ try:
+     headers = {
+               'Content-Type': 'application/json',
+               'Accept': '*/*',
+              }
+     response = requests.post(url, data = json.dumps(data), headers=headers)
+     if(response.ok):
+       print("POST Success", url)
+       if 'ports' in url:
+        valid_response = json.loads(response.text,object_pairs_hook=dict_clean)
+        get_mac_for_ips(valid_response)
+     else:
+       response.raise_for_status()
+ except requests.exceptions.HTTPError as err:
+     print("POST Failed for {} with error".format(url, response.text))
+     print(response.json)
+     print("ERROR",err)
+     raise SystemExit(err)
+
+def get_mac_for_ips(valid_response):
+  ports_info = valid_response["port"]
+  #print(ports_info["mac_address"])
+  #print(ports_info["fixed_ips"][0]["ip_address"])
+  key = ports_info["fixed_ips"][0]["ip_address"]
+  value =  ports_info["mac_address"]
+  ip_mac_db[key] = value
+
+
+def get_httprequest(url):
+  try:
+     response = requests.get(url)
+     if(response.ok):
+       print("GET Success", url)
+       return response.text
+     else:
+       response.raise_for_status()
+  except requests.HTTPError as exception:
+  #except:requests.exceptions.HTTPError as e:
+       print("GET failed for url", url)
+       raise SystemExit(exception)
+
+
+def create_default_segment_table(port):
+  url ='http://localhost:{}/segments/createDefaultTable'.format(port)
+  time.sleep(3)
+  post_httprequest(url)
+
+
+def create_node(ip_mac, port):
+  url= 'http://localhost:{}/nodes'.format(port)
+  data = {}
+  for key, value in ip_mac.items():
+     key_index = list(ip_mac).index(key) if key in ip_mac else None
+     node_name = "node" + str(key_index)
+     node_info = {"local_ip":str(key), "mac_address":str(value), "node_id":node_id[key_index], "node_name":node_name, "server_port":8080, "veth":"eth0"}
+     data["host_info"] = node_info
+     post_httprequest(url, data)
+
+
+def create_vpc(port):
+  network = {}
+  url = 'http://localhost:{}/project/{}/vpcs'.format(port, project_id)
+  network_info = {"admin_state_up":True, "revision_number":0, "cidr":"10.0.1.0/24", "default":True, "description":"vpc", "dns_domain":"domain", "id":network_id, "is_default":True, "mtu":1400, "name":"sample_vpc", "port_security_enabled":True, "project_id":project_id}
```

Comment:
In Python, is it possible to load a web entity class in our main java classes, and create a Python object accordingly? if possible, we don't need to manually update the test case if the java entity gets updated